core/modules/comment/comment.install

```diff
@@ -433,6 +433,1584 @@ function comment_update_1005() {
+            'element_label_class' => '',
+            'element_label_colon' => 1,
+            'element_wrapper_type' => '',
+            'element_wrapper_class' => '',
+            'element_default_classes' => 1,
+            'empty' => '',
+            'hide_empty' => 0,
+            'empty_zero' => 0,
+            'hide_alter_empty' => 1,
+            'text' => '',
+            'link_to_node' => 0,
+          ),
+          'dropbutton' => array(
+            'id' => 'dropbutton',
+            'table' => 'views',
+            'field' => 'dropbutton',
+            'relationship' => 'none',
+            'group_type' => 'group',
+            'ui_name' => '',
+            'label' => 'Operations',
+            'exclude' => 0,
+            'alter' => array(
+              'alter_text' => FALSE,
+              'text' => '',
+              'make_link' => FALSE,
+              'path' => '',
+              'absolute' => FALSE,
+              'external' => FALSE,
+              'replace_spaces' => FALSE,
+              'path_case' => 'none',
+              'trim_whitespace' => FALSE,
+              'alt' => '',
+              'rel' => '',
+              'link_class' => '',
+              'prefix' => '',
+              'suffix' => '',
+              'target' => '',
+              'nl2br' => FALSE,
+              'max_length' => '',
+              'word_boundary' => TRUE,
+              'ellipsis' => TRUE,
+              'more_link' => FALSE,
+              'more_link_text' => '',
+              'more_link_path' => '',
+              'strip_tags' => FALSE,
+              'trim' => FALSE,
+              'preserve_tags' => '',
+              'html' => FALSE,
+            ),
+            'element_type' => '',
+            'element_class' => '',
+            'element_label_type' => '',
+            'element_label_class' => '',
+            'element_label_colon' => 1,
+            'element_wrapper_type' => '',
+            'element_wrapper_class' => '',
+            'element_default_classes' => 0,
+            'empty' => '',
+            'hide_empty' => 0,
+            'empty_zero' => 0,
+            'hide_alter_empty' => 1,
+            'fields' => array(
+              'edit_comment' => 'edit_comment',
+              'approve_comment' => 'approve_comment',
+              'delete_comment' => 'delete_comment',
+              'bulk_form' => 0,
+              'subject' => 0,
+              'name' => 0,
+              'title' => 0,
+              'changed' => 0,
+            ),
+            'destination' => 1,
+          ),
+        ),
+        'header' => array(
+          'result' => array(
+              'id' => 'result',
+              'table' => 'views',
+              'field' => 'result',
+              'relationship' => 'none',
+              'group_type' => 'group',
+              'ui_name' => '',
+              'label' => '',
+              'empty' => 0,
+              'content' => '<em>Number of unapproved comments</em>: @total.',
```

Comment:
```suggestion
              'content' => 'Number of unapproved comments: @total',
```

That period at the end of the count looks weird. Also, we use italics for names of things, or when we need to emphasize things; none of which is the case here.